packages/types/lib/IStatistic.ts

```diff
@@ -0,0 +1,101 @@
+// eslint-disable-next-line
+import type { EventEmitter } from 'events';
+import type { ILink } from '@comunica/bus-rdf-resolve-hypermedia-links';
+import type { ActionContextKey } from '@comunica/core';
+import type { IActionContext } from './IActionContext';
+import type { IQuerySource } from './IQuerySource';
+
+export interface IStatistic {
+  /**
+   * Query string of tracked execution
+   */
+  query: string, 
```

Comment:
This seems odd to have here. Can it be removed?

Reply:
I agree that it's odd, my reasoning was if an engine instance is used for multiple queries in a row and the user wants to track the statistics computed by comunica, a statistics logger might want to output the query associated with the statistic. However, this should be removed if we're not using the logger.